backends/events/sdl/resvm-sdl-events.cpp

```diff
@@ -89,7 +103,8 @@ bool ResVmSdlEventSource::shouldGenerateMouseEvents() {
 		return true;
 	}
 
-	if (g_gui.isActive()) {
+	// FIXME: call to CursorMan.isVisible is not clean ! But don't know how to do better
+	if (g_gui.isActive() || CursorMan.isVisible()) {
```

Comment:
Probably checking if the engine is paused is a little better. In any case there needs to be a comment here explaining that this is to send mouse events when the virtual keyboard is open. It's not obvious at all.

Reply:
Good idea ... I will check in that way !
Thank you.